src/scales/scale.radialLinear.js

```diff
@@ -423,12 +423,17 @@ module.exports = LinearScaleBase.extend({
 	},
 
 	getIndexAngle: function(index) {
-		var angleMultiplier = (Math.PI * 2) / getValueCount(this);
-		var startAngle = this.chart.options && this.chart.options.startAngle ?
-			this.chart.options.startAngle :
-			0;
+		var me = this;
+		var angleMultiplier = (Math.PI * 2) / getValueCount(me);
+		var startAngle = me.chart.options && me.chart.options.startAngle
+			? me.chart.options.startAngle
+			: 0;
+
+		var startAngleRadians = (me.chart.config.type === 'polarArea')
```

Comment:
This works, but it'd be nicer if the scale didn't have to know about the chart type. What if we instead introduce an option for degrees vs radians? Then the scale doesn't need to know which chart is being used and users can specify which ever they prefer

Reply:
Trying to fix a bug here, I'd rather not introduce a new option to work around it.
If this was fixed using a new option, then would have to specify conflicting defaults to chart types. IMO that would be a new confusing public API to keep backward compatible.